test/functional/vimscript/system_spec.lua

```diff
@@ -596,22 +596,21 @@ describe('shell :!', function()
       4
       2]])
     feed(':4verbose %w !sort<cr>')
-    if is_os('win') then
-      screen:expect {
-        any = [[Executing command: .?sort %< .*]],
-      }
+    screen:expect {
+      any = [[Executing command: "sort".*]],
+    }
+    feed('<CR>')
+
+    if not n.has_powershell() then
+      pending('powershell not found, skip powershell part', function() end)
```

Comment:
just `return`. `pending` is for things that require attention in the future. Don't want the "skipped tests" list to grow permanently for things that cannot be fixed.